Jenkinsfile

```diff
@@ -36,6 +36,7 @@ pipeline {
             start /wait cmd /c "C:/Instrument/Apps/EPICS/start_ibex_server.bat"
             call C:/Instrument/Apps/EPICS/config_env.bat
             call C:/Instrument/Apps/Python/python.exe run_tests.py || echo "running tests failed."
+            call C:/Instrument/Apps/EPICS/ISIS/JournalParser/master/test/run_tests.bat || echo "running tests failed."
```

Comment:
As discussed, please remember to rename the repo and Jenkins build to not be "genie_python_system_tests" so as to reflect its true nature.

Reply:
done